src/constructs/stack-set/__snapshots__/stack-set.test.ts.snap

```diff
@@ -78,9 +78,6 @@ exports[`The GuStackSet construct should correctly provision a stack with a stac
             }
           ]
         },
-        "StreamModeDetails": {
-          "StreamMode": "PROVISIONED"
-        },
```

Comment:
This change was introduced in [v2.76.0](https://github.com/aws/aws-cdk/releases/tag/v2.76.0).